packages/react-composites/src/composites/MeetingComposite/SidePane.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+import { ChatComposite, ChatAdapter } from '../ChatComposite';
+import { CommandBarButton, DefaultButton, PartialTheme, Theme, Stack } from '@fluentui/react';
+import {
+  sidePaneCloseButtonStyles,
+  sidePaneContainerStyles,
+  sidePaneHeaderStyles,
+  sidePaneBodyStyles,
+  peopleSubheadingStyle
+} from './styles/SidePane.styles';
+import { ParticipantList, CommunicationParticipant } from '@internal/react-components';
+import copy from 'copy-to-clipboard';
+
+const SidePane = (props: { headingText: string; children: React.ReactNode; onClose: () => void }): JSX.Element => {
+  return (
+    <Stack styles={sidePaneContainerStyles} tokens={{ childrenGap: '0.5rem' }}>
+      <Stack.Item>
+        <Stack horizontal horizontalAlign="space-between" styles={sidePaneHeaderStyles}>
+          <Stack.Item>{props.headingText}</Stack.Item>
+          <CommandBarButton
+            styles={sidePaneCloseButtonStyles}
+            iconProps={{ iconName: 'cancel' }}
+            onClick={props.onClose}
+          />
+        </Stack>
+      </Stack.Item>
+      <Stack.Item styles={sidePaneBodyStyles}>{props.children}</Stack.Item>
+    </Stack>
+  );
+};
+
+export const EmbeddedPeoplePane = (props: {
+  participants: CommunicationParticipant[];
+  myUserId: string;
+  inviteLink?: string;
+  onClose: () => void;
+}): JSX.Element => {
+  const { inviteLink } = props;
+  return (
+    <SidePane headingText={'People'} onClose={props.onClose}>
+      <Stack tokens={{ childrenGap: '0.5rem' }}>
```

Comment:
Similar to how `className={mergeStyles()}` is recommended to be provided as a variable to avoid re-rendering. Is that the case with tokens as well?

Reply:
Great point, my gut reaction is yes as tokens prop will have a new reference, I'll update these

Reply:
Updated 👍